ush/gfs_bufr_netcdf.sh

```diff
@@ -49,7 +49,6 @@ else
 fi
 
 SFCF="sfc"
-CLASS="class1fv3"
```

Comment:
Why did you remove this? It is used later on.

Reply:
Fixed.